src/odb/src/db/dbModule.cpp

```diff
@@ -212,36 +212,56 @@ void dbModule::addInst(dbInst* inst)
   }
 
   _inst->_module = module->getOID();
-  _inst->_module_next = module->_insts;
+
+  if (module->_insts != 0) {
+    _dbInst* _tail = block->_inst_tbl->getPtr(module->_insts);
```

Comment:
Isn't this the current head?  Local variables should not have a lead '_'

Reply:
yes, it is; will fix it.